db/seeds.rb

```diff
@@ -1,7 +1,128 @@
+
+bet2 = Bet.create(bet_description: 'Soccer: England VS Spain',
+                  creator_id: ivan.id,
+                  expires_at: DateTime.tomorrow,
+                  status: 0)
+b2_option1 = BetOption.create(option_text: 'England wins 2:0',
+                              bet: bet2)
+b2_option2 = BetOption.create(option_text: 'Spain wins 4:3',
+                              bet: bet2)
+b2_option3 = BetOption.create(option_text: 'England wins 1:0',
+                              bet: bet2)
+b2_option4 = BetOption.create(option_text: 'Draw',
+                              bet: bet2)
+
+UserBet.create(user: bob,
+               bet: bet2,
+               bet_option: b2_option1,
+               amount_bet: 25.34)
+UserBet.create(user: sarah,
+               bet: bet2,
+               bet_option: b2_option2,
+               amount_bet: 64.24)
+UserBet.create(user: ivan,
+               bet: bet2,
+               bet_option: b2_option3,
+               amount_bet: 34.35)
+UserBet.create(user: maria,
+               bet: bet2,
+               bet_option: b2_option4,
+               amount_bet: 56.99)
+bet2.resolve(winning_option: b2_option4)
+
+# Unresolved bets
+bet3 = Bet.create(bet_description: 'I bet Ivan will not deliver '\
+                  'the project on time',
+                  status: 1,
+                  expires_at: DateTime.tomorrow,
+                  creator_id: sarah.id)
+b3_option1 = BetOption.create(option_text: 'He will',
+                              bet: bet3)
+b3_option2 = BetOption.create(option_text: 'He won\'t',
+                              bet: bet3)
+UserBet.create(bet: bet3,
+               user: bob,
+               bet_option: b3_option2,
+               amount_bet: 13.5)
+UserBet.create(bet: bet3,
+               user: maria,
+               bet_option: b3_option1,
+               amount_bet: 20.99)
+
+# Expired bets
+bet4 = Bet.new
+Timecop.freeze(Date.new(2016, 8, 17)) do
+  bet4 = Bet.create(bet_description: 'Rio Olympics: who will win?',
+                    status: 1,
+                    expires_at: DateTime.new(2016, 8, 20),
+                    creator_id: ivan.id)
+end
+b4_option1 = BetOption.create(option_text: 'USA',
+                              bet: bet4)
+b4_option2 = BetOption.create(option_text: 'GBR',
+                              bet: bet4)
+b4_option3 = BetOption.create(option_text: 'CHN',
+                              bet: bet4)
+UserBet.create(bet: bet4,
+               user: bob,
+               bet_option: b4_option1,
+               amount_bet: 23.32)
+UserBet.create(bet: bet4,
+               user: maria,
+               bet_option: b4_option2,
+               amount_bet: 53.3)
+UserBet.create(bet: bet4,
+               user: ivan,
+               bet_option: b4_option3,
+               amount_bet: 35.2)
+
+# Filler bets
+50.times do |bet|
+  Bet.create(bet_description: "Filler test bet #{bet + 1}",
+             expires_at: DateTime.tomorrow,
```

Comment:
I could add a couple of years to it, but ultimately doesn't matter to me because I tend to drop-migrate the database a lot. Thanks for the suggestion!